drivers/w1/Kconfig.ds2484

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2022, Caspar Friedrich <c.s.w.friedrich@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config W1_DS2484
+	bool "DS2484 Single-Channel 1-Wire Master"
+	depends on I2C
+	depends on DT_HAS_MAXIM_DS2484_ENABLED
```

Comment:
@galak should this be like the #47949 as well?

```
depends on I2C && $(dt_compat_on_bus,$(DT_COMPAT_MAXIM_DS2484),i2c)
```

Reply:
No, that should only be needed if the condition depends on having enabled one of two different buses.
Because there it could happen, that you accidentally enable the wrong bus. I.e. enable I2C but add the device node for SPI.

Reply:
@str4t0m thanks for the enlightenment!